tests/jetty-ee10-test-maven-plugin/src/it/jetty-start-forked-mojo-it/invoker.properties

```diff
@@ -1 +1 @@
-invoker.goals = verify -fae
+invoker.goals = verify -c -fae
```

Comment:
Seems harmless to remove, but perhaps changes such as this should be done in jetty-10 then merged through to jetty-12.

Reply:
The `-c` was an experiment to make the checksum validation a bit lazier.

When I was trying to use `jetty-slf4j-impl` in the IT projects, I kept getting checksum errors from maven.
That's because during the IT tests, there's a fake "remote maven repo" that is actually just pointing to the local repo.
If a project from local is built and installed with `mvn install`, there are no checksums produced and included in that local repo.
This breaks maven as the fake remote maven repo doesn't have the required checksums.
Unfortunately there's no way to add back the checksums during the `install`, nor is there a way to totally disable checksum checks in maven via command line.

Reply:
Yes, I've seen those checksum errors during the it tests, but the test works anyway. If the property is useless, remove it, but make sure it's removed from _all_ of the `invoker.properties` files.

Reply:
Incorrect.

The checksum errors I encountered were fatal.
This was because I attempted to add `jetty-slf4j-impl` in the IT projects.
That results in NO checksums present.
That's a fatal scenario for the build/test of the IT projects.

If I added something that isn't part of the reactor, and the checksums exist, but were bad, then it's non-fatal.

Reply:
Removed in 0128910c5e7afb24ab85a03c1708cbbc407a91c3